src/alchemlyb/estimators/mbar_.py

```diff
@@ -152,6 +152,14 @@ class AutoMBAR(MBAR):
     :class:`AutoMBAR` may be useful in high-throughput calculations where it can avoid 
     failures due non-converged MBAR estimates.
 
+    Parameters
+    ----------
+
+    method : str, optional, default=None
+        The optimization routine to use. This parameter is default to None.
+        When a specific method is set, AutoMBAR will behave in the same ways
+        as MBAR.
```

Comment:
Make this important addition very explicit
```suggestion
        as MBAR.

        .. versionadded:: 1.0.0
        
```

Reply:
Add entry to CHANGELOG